nft_ingester/src/index_syncronizer.rs

```diff
@@ -74,38 +77,60 @@ where
         rx: &tokio::sync::broadcast::Receiver<()>,
         run_full_sync_threshold: i64,
     ) {
-        if let Ok(SyncStatus::FullSyncRequired(_)) =
-            self.get_sync_state(run_full_sync_threshold).await
-        {
-            let res = self.full_syncronize(rx).await;
-            match res {
-                Ok(_) => {
-                    tracing::info!("Full synchronization finished successfully");
-                }
-                Err(e) => {
-                    tracing::error!("Full synchronization failed: {:?}", e);
+        let asset_types = [AssetType::Fungible, AssetType::NonFungible];
+        for asset_type in asset_types {
+            if let Ok(SyncStatus::FullSyncRequired(_)) = self
+                .get_sync_state(run_full_sync_threshold, asset_type)
+                .await
+            {
+                let res = self.full_syncronize(rx).await;
```

Comment:
running 2 full syncs consequentially is not the best idea given each will take hours on production system

Reply:
Fixed